packages/components/kbd/src/Kbd.styles.tsx

```diff
@@ -0,0 +1,5 @@
+import { cva } from 'class-variance-authority'
+
+export const kbdStyles = cva([
+  'border-sm border-b-md bg-neutral-container ps-md pe-md border-outline whitespace-nowrap rounded-sm font-mono text-[0.8em] font-bold leading-none',
```

Comment:
"text-[0.8em]" is not a typography token of spark. 

I think we should stick to spark tokens. Also text is not vertically centered (maybe because no line-Height is set as it is an hardcoded value and not a token ?)
![Capture d’écran 2023-06-09 à 10 14 27](https://github.com/adevinta/spark/assets/2033710/d1d111e3-45f0-4a85-8eea-7be913265cba)

Reply:
`leading-3`?

Reply:
I don't think we have a font-size named like that. Maybe `caption` or `small` ?